validation/td-json-schema-validation.json

```diff
@@ -1141,15 +1131,49 @@
                             ]
                         },
                         "flow": {
-                            "type": "string",
-                            "enum": [
-                                "code"
-                            ]
+                           "oneOf": [
```

Comment:
Actually it doesn't work since `code` would validate both of the schemas: https://www.jsonschemavalidator.net/s/VVtuuHK5

Does `anyOf` work with Intellisense

Reply:
Thanks, good point! I've just checked with `anyOf` and it seems to produce the same TypeScript code. Works well with Intellisense too. Updating...